### PR TITLE
add access button to the BV

### LINKED ISSUE #64: Add an access to the Batch Viewer on order page
As Anxo mentioned in the [doc for the reiteration on BV,](https://docs.google.com/document/d/12tMl4Cn0gh7rKGJjAjsn4kuQsDu9wfAl_eR5AgDJgsM/edit#) we need to add an access to the BV in the "order details" page.

Draft proposal from Anxo:
image

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Closes #64 
image
image

# To Test

1. <<Step one>> Open the page [`Order detail`](https://pr73--gpui.review.gnosisdev.com/orders/0x50cf369622057541b519eede5cd5c58c33e9fb66a8ac9c4419e1cc708be0351d79d2e55e30cd4183715f53517d5c5de850aba2cb6192814e) 
Check the appearance of the button "View batch graph". Later we will be adding a link to the specific batch.

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]
LGTM!

**Comment by elena-zh** [comment]
@alongoni , the only thing that I noticed here is that changes for the 'Copy' feature from #68 PR are not applied here

File: src/components/orders/DetailsTable/index.tsx [hunk 1]
```diff
@@ -170,11 +211,16 @@ export function DetailsTable(props: Props): JSX.Element | null {
...
-                    onCopy={(): void => onCopy('settlementTx')}
-                    contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
-                  />
+                  <Wrapper>
+                    <RowWithCopyButton
+                      textToCopy={txHash}
+                      onCopy={(): void => onCopy('settlementTx')}
+                      contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
+                    />
+                    <Button role="button">
```
Comment (ramirotw) [inline src/components/orders/DetailsTable/index.tsx:220]: This should trigger a route change so it makes sense for it be a Link and it should point to `to={`/tx/${txHash}/batch-graph`}`

**Review by ramirotw — CHANGES_REQUESTED** [review]
Almost there, and please point the base branch to `41-anchor-url` which is @henrypalacios's branch

File: src/components/orders/DetailsTable/index.tsx (hunk 1 above)
Comment (ramirotw) [reply src/components/orders/DetailsTable/index.tsx:220]: Update: url according to https://github.com/cowprotocol/explorer/pull/49 is `/tx/${txHash}/?tab=graph`

**Review by ramirotw — CHANGES_REQUESTED** [review]
could we move all the changes in the links and package.json to a different PR?

File: src/components/orders/DetailsTable/index.tsx [hunk 2]
```diff
@@ -170,11 +211,18 @@ export function DetailsTable(props: Props): JSX.Element | null {
...
-                    textToCopy={txHash}
-                    onCopy={(): void => onCopy('settlementTx')}
-                    contentsToDisplay={<LinkWithPrefixNetwork to={`/tx/${txHash}`}>{txHash}</LinkWithPrefixNetwork>}
-                  />
+                  <Wrapper>
+                    <RowWithCopyButton
+                      textToCopy={txHash}
+                      onCopy={(): void => onCopy('settlementTx')}
+                      contentsToDisplay={
+                        <LinkWithPrefixNetwork to={`/tx/${txHash}/?tab=graph`}>{txHash}</LinkWithPrefixNetwork>
```
Comment (ramirotw) [inline src/components/orders/DetailsTable/index.tsx:219]: Sorry if I wasn't clear, this link shouldn't be changed, I was referring to the `View batch graph` button that needs to be a link that points to `/tx/${txHash}/?tab=graph`

**Review by ramirotw — CHANGES_REQUESTED** [review]
nice!! just 1 little thing and it's good to go.

Remove the underline during on-hover

image

**Review by ramirotw — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @alongoni , I notices this weird issue in this PR
image

It is reproducible when open TX list first, then to switch to the BV. See the video: https://watch.screencastify.com/v/15HqO9IiqlC1DV1dOTbt

But I was not able to reproduce the issue in #49 PR , so seems like it is fixed there. 
But could you please take a look and confirm that there is no need to worry about it? 

Thanks!

**Comment by ramirotw** [comment]
@elena-zh that's correct, the base branch for this PR was set to @henrypalacios' branch, but when this PR was merged the issue you are pointing out was still present (now it's fixed)

**Pull request merged**